examples/feature_alignment_example/client.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from logging import INFO
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader, TensorDataset
+
+from examples.models.logistic_regression import LogisticRegression
+from fl4health.clients.tabular_data_client import TabularDataClient
+from fl4health.utils.metrics import Accuracy, Metric
+
+
+class Mimic3TabularDataClient(TabularDataClient):
+    def __init__(
+        self, data_path: Path, metrics: Sequence[Metric], device: torch.device, id_column: str, target_column: str
+    ) -> None:
+        super().__init__(data_path, metrics, device, id_column, target_column)
+
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        indices = np.random.permutation(self.aligned_features.shape[0])
+        shuffled_data = self.aligned_features[indices]
+        shuffled_targets = self.aligned_targets[indices]
+        split_percentage = 0.9
+        split_point = int(shuffled_data.shape[0] * split_percentage)
+
+        train_data = shuffled_data[:split_point]
+        val_data = shuffled_data[split_point:]
+        train_targets = shuffled_targets[:split_point]
+        val_targets = shuffled_targets[split_point:]
+
+        tensor_train_data = torch.from_numpy(train_data.toarray())
+        tensor_train_targets = torch.from_numpy(train_targets)
+        tensor_val_data = torch.from_numpy(val_data.toarray())
+        tensor_val_targets = torch.from_numpy(val_targets)
+
+        tensor_train_data = tensor_train_data.float()
```

Comment:
Rather than storing the intermediate `tensor_train_data` and then converting, I might recommend just taking the `.float()` onto the conversion above, since its pretty simple and straightforward what's happening and reduces lines.